File: 03.Functions-And-Statements-More-Exercise/carWash.js
function washCar(arrayOfActions) {
    let value = 0

    for (const action of arrayOfActions) {
        if (action === 'soap') {
            value += 10
        } else if (action === 'water') {
            value *= 1.20
        } else if (action === 'vacuum cleaner') {
            value *= 1.25
        } else if (action === 'mud') {
            value *= 0.90
        } 
    }

    console.log(`The car is ${value.toFixed(2)}% clean.`)
}

washCar(['soap', 'soap', 'vacuum cleaner', 'mud', 'soap', 'water'])
washCar(["soap", "water", "mud", "mud", "water", "mud", "vacuum cleaner"])

File: 03.Functions-And-Statements-More-Exercise/car_wash.py
def wash_car(list_of_actions):
    value = 0

    for action in list_of_actions:
        if action == "soap":
            value += 10
        elif action == "water":
            value *= 1.20
        elif action == "vacuum cleaner":
            value *= 1.25
        elif action == "mud":
            value *= 0.90

    print(f"The car is {value:.2f}% clean.")


wash_car(['soap', 'soap', 'vacuum cleaner', 'mud', 'soap', 'water'])
wash_car(["soap", "water", "mud", "mud", "water", "mud", "vacuum cleaner"])